draft-ietf-quic-transport.md

```diff
@@ -2304,13 +2304,25 @@ the application requests that the connection be closed.  The application
 protocol can use an CONNECTION_CLOSE frame with an appropriate error code to
 signal closure.
 
-If the connection has been successfully established, endpoints MUST send any
-CONNECTION_CLOSE frames in a 1-RTT packet.  Prior to connection establishment a
-peer might not have 1-RTT keys, so endpoints SHOULD send CONNECTION_CLOSE frames
-in a Handshake packet.  If the endpoint does not have Handshake keys, or it is
-not certain that the peer has Handshake keys, it MAY send CONNECTION_CLOSE
-frames in an Initial packet.  If multiple packets are sent, they can be
-coalesced (see {{packet-coalesce}}) to facilitate retransmission.
+When sending CONNECTION_CLOSE, the goal is to ensure that a peer will process
+the frame.  Generally, this means sending the frame in a packet with the highest
+level of packet protection to avoid the packet being discarded.  However, during
+the handshake, it is possible that more advanced packet protection keys are not
+available to a peer, so the frame MAY be replicated in a packet that uses a
+lower packet protection level.
```

Comment:
This paragraph doesn't really say anything that the next paragraph doesn't cover. Also, `a peer` sounds like an endpoint can have multiple peers.

Reply:
I missed this earlier, but I agree. I'm fine with dropping this paragraph entirely. It seems like it's simply repeating what's in the next para, but in more general terms.

Reply:
I don't have a strong opinion, but I see your point, this was @martinthomson addition, so I'll let him comment.

Reply:
The point was to articulate the principles behind the design.  It might be that the mechanics are sufficiently precise that that is not necessary, but we did have #2151.  @ianswett's changes didn't really directly address that.

Reply:
If people want to fix this, please open a followup issue.